tensorbay/cli/tag.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Implementation of gas tag."""
+import sys
+from typing import Dict
+
+import click
+
+from ..client.gas import DatasetClientType
+from .tbrn import TBRN, TBRNType
+from .utility import get_dataset_client, get_gas
+
+
+def _implement_tag(obj: Dict[str, str], tbrn: str) -> None:
+    info = TBRN(tbrn=tbrn)
+
+    if info.type != TBRNType.DATASET:
+        click.echo(f'To operate a tag, "{info}" must be a dataset', err=True)
+        sys.exit(1)
+
+    gas = get_gas(**obj)
+    dataset_client = get_dataset_client(gas, info)
+
+    _list_tags(dataset_client)
+
+
+def _list_tags(dataset_client: DatasetClientType) -> None:
+    tags = dataset_client.list_tags()
+    for tag in tags:
```

Comment:
```python
for tag in dataset_client.list_tags():
    ...
```

Reply:
After modification, the body of function only has two lines, is it better to write directly this section in _implement_tag() instead of additional function of _list_tags()?

Reply:
command `gas tag` not only list tags, but also create and delete.
use functions to separate them make it easy to understand.
Also there is no limitation about the minimum number of lines in a function.